### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by XXJJXJ**

### CODE REVIEW

**Comment by kanjitp** [comment]
👀

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 1]
```diff
@@ -26,20 +24,6 @@
 
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
-    @Test
-    public void execute_validIndexUnfilteredList_success() {
-        Participant participantToDelete = model.getFilteredParticipantList()
```
Comment (kanjitp) [inline src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:31]: pass for me

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 2]
```diff
@@ -48,23 +32,6 @@ public void execute_invalidIndexUnfilteredList_throwsCommandException() {
         assertCommandFailure(deleteCommand, model, Messages.MESSAGE_INVALID_PARTICIPANT_DISPLAYED_INDEX);
     }
 
-    @Test
-    public void execute_validIndexFilteredList_success() {
-        showParticipantAtIndex(model, INDEX_FIRST_PARTICIPANT);
```
Comment (kanjitp) [inline src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:53]: pass for me

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 3]
```diff
@@ -33,20 +33,6 @@
 
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
-    @Test
-    public void execute_allFieldsSpecifiedUnfilteredList_success() {
-        Participant editedParticipant = new ParticipantBuilder().build();
```
Comment (kanjitp) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:38]: pass for me

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 4]
```diff
@@ -79,26 +65,6 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
-    @Test
-    public void execute_filteredList_success() {
-        showParticipantAtIndex(model, INDEX_FIRST_PARTICIPANT);
```
Comment (kanjitp) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:84]: I give up, pass for me 😢

Comment (XXJJXJ) [reply src/test/java/seedu/address/logic/commands/EditCommandTest.java:84]: They all pass locally for me too. But not Github 😰

**Review by ntwbruce — APPROVED** [review]
LGTM

**Pull request merged**